transpiler/src/target/typescript/core/components/integration-event/integrationEventParameter.ts

```diff
@@ -34,10 +48,29 @@ export const integrationEventParameterToTargetLanguage = (
     boundedContext: boundedContextModule.boundedContextName.wordsWithSpaces,
     module: boundedContextModule.moduleName.wordsWithSpaces,
   };
-  const dependencies = getChildDependencies(integrationEventIdentifier, contextData);
+  let dependencies = [];
+  if (contextData.boundedContext === BITLOOPS_BOUNDED_CONTEXT) {
+    const packageForModule = packagesEventsMappping[contextData.module];
+    if (!packageForModule) {
+      throw new Error(`Package ${contextData.module} not found`);
+    }
+
+    dependencies = INTEGRATION_EVENT_PARAMETER_DEPENDENCIES(
+      integrationEventIdentifier,
+      packageForModule,
+    );
+  } else {
+    dependencies = getChildDependencies(integrationEventIdentifier, contextData);
+  }
 
   return {
     output: `${value}:${integrationEventIdentifier}`,
     dependencies,
   };
 };
+
+const packagesEventsMappping = {
+  authNestPassport: '@bitloops/bl-boilerplate-infra-nest-auth-passport',
+  nestJetsream: '@bitloops/bl-boilerplate-infra-nest-jetsream',
+  telemetry: '@bitloops/bl-boilerplate-infra-telemetry',
+};
```

Comment:
These maybe should be moved somewhere else in order for anyone to reuse them if needed?